railties/CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+*   Ensure Rails migration generator respects system-wide primary key config
+
+    When rails is configured to use a specific primary key type:
+    ```
+    config.generators do |g|
+      g.orm :active_record, primary_key_type: :uuid
+    end
+    ```
+
+    Previously:
+
+        $ bin/rails g migration add_location_to_users location:references
+
+    The references line in the migration would not have `type: :uuid`.
+    This change causes the type to be applied appropriately.
+    GH#23422
```

Comment:
Can you remove `GH#23422` - we don't normally reference issues in the CHANGELOG.